### PR TITLE
make mathias shaw not suck

### PR DESCRIPTION
**Pull request by edmund-h**
## Changelog:
- switched his intrigue and learning
- switched out two of his traits to not nerf his intrigue skill

## Tests:
- start as Varian in 605, your spymaster should have an intrigue of 17

- [ ] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log` except `portrait_decals.cpp:101`
- [ ] The mod takes less than 5.5 GB in the Task Manager (Windows)

# How to test:

### CODE REVIEW

**Comment by Grekopithikos** [comment]
Is the vengeful trait fitting for his personality?

File: history/characters/1_azerothian.txt [hunk 1]
```diff
@@ -13158,8 +13156,6 @@
 			set_variable = { name = wc_dexterity_physical_lifestyle_additional_perks_variable value = wc_perks_needed_for_level_4_physical_trait_value }
 			set_variable = { name = wc_strength_physical_lifestyle_additional_perks_variable value = wc_perks_needed_for_level_4_physical_trait_value }
 		}
-		# trait=physical_lifestyle_dexterity_4
-		# trait=physical_lifestyle_strength_4
```
Comment (Grekopithikos) [inline history/characters/1_azerothian.txt:13162]: I don't think you ought to remove those

Comment (edmund-h) [reply history/characters/1_azerothian.txt:13162]: ok, doesnt look like this system is being used at all. idk how it made it into production. anyway i will readd them

**Review by Grekopithikos — APPROVED** [review]
looks good to me

**Pull request merged**